client/my-sites/google-my-business/stats/index.js

```diff
@@ -66,9 +66,22 @@ class GoogleMyBusinessStats extends Component {
 		} );
 	};
 
-	renderViewsTooltipForDatanum = ( datanum, dataSeries ) => {
+	renderViewsTooltipForDatanum = ( datanum, interval ) => {
 		const { value: valueCount, date } = datanum;
-		if ( dataSeries.length > 20 ) {
+		if ( interval === 'quarter' ) {
+			return this.props.translate(
+				'%(value)d view on week %(week)s',
+				'%(value)d views on week %(week)s',
+				{
+					count: valueCount,
+					args: {
+						value: valueCount,
+						week: moment( date ).format( 'D MMMM YYYY' ),
```

Comment:
check out `LL` as a localized format that will update with the language change. in some languages this given format of `D MMMM YYYY` may appear strange.